leetcode/math.md

```diff
@@ -0,0 +1,24 @@
+# Math
```

Comment:
# Invalid PULL Commits

** Invalid Commit Message: LEETCODE 1022 Refactored base-7 **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Merge remote-tracking branch 'origin/base-7' into base-7

# Conflicts:
#	leetcode/math.md **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']